eox_tagging/test/test_models.py

```diff
@@ -422,3 +422,9 @@ def test_validation_date_not_in_list(self):
                 expiration_date=datetime.datetime(2020, 4, 19, 10, 20, 30),
                 activation_date=datetime.datetime(2020, 6, 16, 10, 20, 30),
             )
+
+    def test_soft_delete_queryset(self):
+        """Used to test deactivating tags using a tag queryset."""
+        Tag.objects.delete()
```

Comment:
mmm could you do this with a instanced class and verify the inactivated_at datetime

Reply:
Yes! I just did!
https://github.com/eduNEXT/eox-tagging/blob/14c65c92413588632af31ca14828d2a82aa21980/eox_tagging/test/test_models.py#L258